test/chrome/script.py

```diff
@@ -0,0 +1,58 @@
+# Run Chromium tests for HTTPS Everywhere
+#
+# This script may be executed as `python script.py [directory of CRX]`
+#
+# The script is compatible with Python 2. Python 3 is not tested.
+# Selenium, WebDriver and Google Chrome (or Chromium) must be installed
+# in order for the script to run successfully. A desktop version
+# of linux is required for the script to run correctly as well.
+# Otherwise, use pyvirtualdisplay.
+
+import time, sys, glob, os, traceback
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions
+from selenium.webdriver.common.by import By
+
+class bcolors:
+	HEADER = '\033[95m'
+	OKBLUE = '\033[94m'
+	OKGREEN = '\033[92m'
+	WARNING = '\033[93m'
+	FAIL = '\033[91m'
+	ENDC = '\033[0m'
+	BOLD = '\033[1m'
+	UNDERLINE = '\033[4m'
+
+extension = None
+
+for file in os.listdir(sys.argv[1]):
+	if file.endswith(".crx"):
+		extension = file
+
+if(extension == None):
+	try:
+		raise Exception()
+	except:
+		traceback.print_exc(file=sys.stdout)
+		exit(3)
+
+print '' #New line
```

Comment:
I'm not sure what should be done when something fails. My first thought was "just raise an exception".